cohorts/views.py

```diff
@@ -26,6 +26,8 @@
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth.models import User
 from django.contrib.auth.models import User as Django_User
+from django.conf import settings
```

Comment:
import settings is not used, so didn't need to be added and import messages was already imported